### DISCUSSION TITLE
Black screen while launching Ghost on Tsushima through Lutris

### DISCUSSION THREAD

**Discussion by ConsularParadi** [games-on-whales/wolf#155] [Q&A] [answered]
Hi, thank you for adding support for lutris by default.
I have been setting up a remote nvidia RTX 5000 for gaming. I was able to setup and install games on whales with lutris but whenever I launch the game all I see is a black screen while the audio of the game can be heard.
When launching it shows an error that `the graphics card would work better with Nvidia driver version 552.xx but current version is 535.xx`. I have updated my `ubuntu 22.04` with nvidia drivers 550 because that is the max that seems to working on the machine but the error remains the same.
I am using ssh to run the Wolf docker container.
Is there something I am missing ? Like rebuilding docker containers etc.

**Comment by BranislavF** [comment, marked as answer]
You need to set this game for Exclusive Fullscreen. That helped me. By the way - is Lutris working for you by default ? Epic Games Store is not - at least not for me.

**Reply by ConsularParadi** [reply]
Hi @BranislavF, thanks for the info. Couldn't have figured this out on my own.
I have not installed Epic Games Store but Lutris works just fine. Tested on both rtx 5000 and geforce 950m.

**Reply by BranislavF** [reply]
> Hi @BranislavF, thanks for the info. Couldn't have figured this out on my own. I have not installed Epic Games Store but Lutris works just fine. Tested on both rtx 5000 and geforce 950m.

Lutris itself is running fine, but as soon as you will try some launcher, it can be a problem :) But anyway, I am glad that the Ghost of Tsushima fix helped you. Happy gaming :)
